package/firmware/amd64-microcode/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=amd64-microcode
-PKG_VERSION:=20180524
+PKG_VERSION:=20191218
```

Comment:
Henrique de Moraes Holschuh from Debian warned us to take version 20191021, see here:
https://www.mail-archive.com/openwrt-devel@lists.openwrt.org/msg49810.html

I do not know if the problem he is talking about is fixed now.

Reply:
It looks like 20191218 removed one microcode update which caused problems:
https://metadata.ftp-master.debian.org/changelogs//non-free/a/amd64-microcode/amd64-microcode_3.20191218.1_changelog
I think this is ok.

Reply:
Yes, the microcode of zen2 processor structure is not included in 20191218